src/simplewallet/simplewallet.cpp

```diff
@@ -187,9 +146,10 @@ namespace
     std::cout << prompt;
 
     std::string buf;
+#ifdef _WIN32
+    buf = tools::input_line_win();
```

Comment:
Probably should have emulated std::getline() semantics instead, e.g.
  `tools::getline(buf);`

Reply:
This paradigm is actually faster in C++11 onwards (copy elision).